### PR TITLE
Initial cleanup part 2

### PR DESCRIPTION
**Pull request by robertaboukhalil**
This PR is to:
- Bring over priority queue from SplitThreader
- Bring in SplitThreader tests
- Bring in latest d3-supertable from SplitThreader to fix console errors
- Replace `saveSvgAsPng.js` library and support SVG
- Clean up build script

### CODE REVIEW

File: js/index_ribbon.js [hunk 1]
```diff
@@ -584,25 +584,23 @@ function get_name() {
...
 }
 
-function screenshot_top() {
-  saveSvgAsPng(
-    document.getElementById("svg_multi_read"),
-    get_name() + "_multi-read.png",
-    { scale: 4 }
-  );
+async function screenshot_top() {
+  await exportViz({
```
Comment (robertaboukhalil) [inline js/index_ribbon.js:588]: Supports SVG too but not exposed in the UI

File: js/index_ribbon.js [hunk 2]
```diff
@@ -6759,14 +6757,6 @@ function screenshot_individual_reads() {
   load_next_read();
 }
 
-function download(filename, text) {
```
Comment (robertaboukhalil) [inline js/index_ribbon.js:6762]: Moved to `utils.js` file

File: bin/build.sh [hunk 3]
```diff
@@ -1,7 +1,32 @@
 #!/bin/bash
-
 # Hacky bundler script to generate CSS and JS bundles.
-# Note that we use `awk 1` instead of `cat` so we append an extra line after each file. Otherwise,
-# this could break if one of the modules doesn't have a break line at the end of the min.js files.
-awk 1 node_modules/{bootstrap/dist/css/bootstrap.min.css,jquery-ui-dist/jquery-ui.min.css} > css/bundle.css
-awk 1 node_modules/{d3/d3.min.js,jquery/dist/jquery.min.js,bootstrap/dist/js/bootstrap.min.js,@biowasm/aioli/dist/aioli.js,pako/dist/pako.min.js,moment/min/moment.min.js,jquery-ui-dist/jquery-ui.min.js,papaparse/papaparse.min.js} > js/bundle.js
+
```
[the remark this answers is not in the document]
Comment (MariaNattestad) [reply bin/build.sh:3]: Yes!

File: index.html [hunk 4]
```diff
@@ -834,13 +834,6 @@ <h4 class="panel-title">
 														<button id="submit_filters">Submit</button>
 													</td>
 												</tr>
-												<tr><td colspan="2"><hr><label>Downloads:</label></td></tr>
```
[the remark this answers is not in the document]
Comment (MariaNattestad) [reply index.html:837]: Ah yes, correct!

File: js/index_ribbon.js (hunk 1 above)
Comment (MariaNattestad) [reply js/index_ribbon.js:588]: So we'll expose it later, cool!

**Review by MariaNattestad — APPROVED** [review]

**Pull request merged**